arex-storage-web-api/src/main/java/com/arextest/storage/service/config/AbstractConfig.java

```diff
@@ -9,22 +9,35 @@
  */
 public abstract class AbstractConfig {
 
-    protected abstract String getConfigAsString(String key);
+  protected abstract String getConfigAsString(String key);
 
-    public String getConfigAsString(String key, String defaultValue) {
-        return StringUtils.defaultString(getConfigAsString(key), defaultValue);
+  public String getConfigAsString(String key, String defaultValue) {
+    return StringUtils.defaultString(getConfigAsString(key), defaultValue);
+  }
+
+  public int getConfigAsInt(String key, int defaultValue) {
+    String value = getConfigAsString(key);
+    if (!StringUtils.isNumeric(value)) {
+      return defaultValue;
+    }
+
+    try {
+      return Integer.parseInt(value);
+    } catch (NumberFormatException e) {
+      return defaultValue;
     }
+  }
 
-    public int getConfigAsInt(String key, int defaultValue) {
-        String value = getConfigAsString(key);
-        if (!StringUtils.isNumeric(value)) {
-            return defaultValue;
-        }
-
-        try {
-            return Integer.parseInt(value);
-        } catch (NumberFormatException e) {
-            return defaultValue;
-        }
+  public boolean getConfigAsBoolean(String key, boolean defaultValue) {
+    String value = getConfigAsString(key);
+    if (StringUtils.isBlank(value) || !isBoolean(value)) {
+      return defaultValue;
     }
+    return Boolean.parseBoolean(value);
```

Comment:
isBoolean has stipulated that only true or false can be passed in, so is Boolean.parseBoolean redundant?